### DISCUSSION TITLE
What is the purpose of adding a certain time to a transaction? Is it possible to change the pie chart on the transactions menu?

### DISCUSSION THREAD

**Discussion by Jonas3110** [firefly-iii/firefly-iii#9452] [Firefly III] [answered]
Hello guys!

I really enjoy Firefly III so far; it's really a great application! Now, as I've really started to stick with it more, I've noticed some things:

## 1. What is the purpose of adding a specific time and not only a date to a transaction?
Right now, I always write dd.mm.yyyy 00:00 or 12 pm since it doesn't seem to make a difference to add a time. The graph with all my accounts on the overview page doesn't change if I enter a different time on several transactions. 

For example: recently, my accounts had great fluctuations within one day, but the graph doesn't depict that because it doesn't seem to consider the time, only the days.
Am I right that the time really doesn't do anything, or am I missing something? If I am right, is this feature planned? The possibility to fine-tune the graph and see a more precise history would be great.
## 2. Is it possible to change the pie chart on the transactions menu?
When within a month a lot of transactions to different accounts happen and these are also tagged with different categories, the pie chart gets very messy, and smaller transactions tend to vanish under huge transactions. 

Is there a possibility to change the chart or at least the size of the pie chart to see all the different accounts/categories?

**Comment by JC5** [comment, marked as answer]
Hey, great questions.

1. Many users keep track of the moment a transaction is made, regardless of whether it is represented in charts. First, it defines the order in which transactions appear in the lists, and second, because it is data they wish to preserve.
2. No, not yet. But pie charts will be removed from the new layout for precisely this reason.

**Reply by Jonas3110** [reply]
Okay I understand, thanks!

It would be great to see the time additionally shown in the graph although I can imagine it might be a little bit of a hassle. 

Is there a way to make a feature request in general and if so where? In the README 
some of the links under "Contributing" are leading to a 404.

**Reply by JC5** [reply]
Hey, 

Nice find, the link should be fixed.
